docs/tutorials/03_authenticated_user.md

```diff
@@ -0,0 +1,98 @@
+This tutorial assumes you have already followed:
+
+- [Hello World Tutorial]{@tutorial 01_hello_world}
+- [Ephemeral User Tutorial]{@tutorial 02_ephemeral_user}
+
+Complete code for this tutorial is available at: {@link https://github.com/ForstaLabs/forsta-messenger-client/tree/master/examples/authenticated_user}
+
+In the last tutorial we configured the client to use an ephemeral user who is
+created and logged in based on the org ephemeral token. However in production
+systems we recommend creating Forsta users using the Forsta Atlas API and 
+libraries. In this tutorial we will discuss how to connect an authenticated user.
+
+You can create a user using either:
+
+- [Forsta Console]{@link https://app.forsta.io/people}
+- [Atlas API]{@link https://atlas.forsta.io/doc/}
+
+For a user to authenticate with the Forsta Messenger Client they need to provide
+a Forsta generated JWT (JSON Web Token). In a production system JWTs are generated
+by authenticating a user with a saved auth token. For testing purposes you can
+also generate a JWT with the following shell scripts:
```

Comment:
The word generate is misleading here.  It makes me think the user is creating it on their side of the pond.  They are *getting* a JWT from the server.  It's more akin to checking out authentication temporarily as they are short lived and are thrown away after some period of time.

Reply:
Fixed and moved to an external